specification/source/codeGeneration.rst

```diff
@@ -104,19 +104,25 @@ Translation of a Control Sequence using a JSON Intermediate Format
 
 Control companies that choose to not use C-code generation or the FMI standard to
 execute CDL-compliant control sequences can develop translators from
-CDL to their native language. To aid in this process, a CDL to JSON translator
-can be used. Such a translator is currently being developed at
-https://github.com/lbl-srg/modelica-json.
-This translator parses CDL-compliant control sequences to a JSON format.
-The parser generates the following output formats:
-
-1. A JSON representation of the control sequence,
-2. a simplified version of this JSON representation, and
-3. an html-formated documentation of the control sequence.
+CDL to their native language. To aid in this process, an intermediate 
+Control eXchange Format (CXF) can be used. CXF is a JSON-LD representation
+of a CDL sequence, serialized in JSON. `Modelica-json <https://github.com/lbl-srg/modelica-json>`_
+is a reference implementation of such a CDL to CXF translator. 
```

Comment:
The translator is not a "reference implementation" as the translator is not part of the standard. Just write it is a translator.